modules/videoio/src/cap.cpp

```diff
@@ -76,7 +76,7 @@ VideoCapture::VideoCapture(const String& filename, int apiPreference) : throwOnF
 }
 
 VideoCapture::VideoCapture(const String& filename, int apiPreference, const std::vector<int>& params)
-    : throwOnFail(true)
+    : throwOnFail(false)
```

Comment:
Disabled exceptions from constructor again:
- there is no API yet to propagate plugins exceptions
- keep similar behavior between constructors